test/aircraft/AircraftModel.spec.js

```diff
@@ -254,84 +258,91 @@ ava('.isEstablishedOnGlidepath() returns true when an acceptable distance below
     const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
     const isEstablishedOnCourse = true;
     const distanceToDatum = AIRPORT_CONSTANTS.FINAL_APPROACH_FIX_DISTANCE_NM + 1;
-
-    sandbox.stub(model, 'isEstablishedOnCourse', () => isEstablishedOnCourse);
-    sandbox.stub(model.positionModel, 'distanceToPosition', () => distanceToDatum);
-
+    const isEstablishedOnCourseStub = sinon.stub(model, 'isEstablishedOnCourse').returns(isEstablishedOnCourse);
+    const distanceToPositionStub = sinon.stub(model.positionModel, 'distanceToPosition').returns(distanceToDatum);
     const result = model.isOnFinal();
 
     t.false(result);
+
+    isEstablishedOnCourseStub.restore();
+    distanceToPositionStub.restore();
 });
 
 ava('.isOnFinal() returns false when within the final approach fix distance but not on the selected course', (t) => {
     const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
     const isEstablishedOnCourse = false;
     const distanceToDatum = AIRPORT_CONSTANTS.FINAL_APPROACH_FIX_DISTANCE_NM;
-
-    sandbox.stub(model, 'isEstablishedOnCourse', () => isEstablishedOnCourse);
-    sandbox.stub(model.positionModel, 'distanceToPosition', () => distanceToDatum);
-
+    const isEstablishedOnCourseStub = sinon.stub(model, 'isEstablishedOnCourse').returns(isEstablishedOnCourse);
+    const distanceToPositionStub = sinon.stub(model.positionModel, 'distanceToPosition').returns(distanceToDatum);
     const result = model.isOnFinal();
 
     t.false(result);
+
+    isEstablishedOnCourseStub.restore();
+    distanceToPositionStub.restore();
 });
 
 ava('.isOnFinal() returns true when both on the selected course and within the final approach fix distance', (t) => {
     const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
     const isEstablishedOnCourse = true;
     const distanceToDatum = AIRPORT_CONSTANTS.FINAL_APPROACH_FIX_DISTANCE_NM;
-
-    sandbox.stub(model, 'isEstablishedOnCourse', () => isEstablishedOnCourse);
-    sandbox.stub(model.positionModel, 'distanceToPosition', () => distanceToDatum);
-
+    const isEstablishedOnCourseStub = sinon.stub(model, 'isEstablishedOnCourse').returns(isEstablishedOnCourse);
+    const distanceToPositionStub = sinon.stub(model.positionModel, 'distanceToPosition').returns(distanceToDatum);
     const result = model.isOnFinal();
 
     t.true(result);
+
+    isEstablishedOnCourseStub.restore();
+    distanceToPositionStub.restore();
 });
 
-ava('._calculateArrivalRunwayModelGlideslopeAltitude() returns arrival runway\'s glideslope altitude abeam the specified position', (t) => {
-    const model = new AircraftModel(ARRIVAL_AIRCRAFT_INIT_PROPS_MOCK);
-    const expectedResult = 3994.129742601768;
+// TODO: this test is suddenly failing entirely
+// ava('._calculateArrivalRunwayModelGlideslopeAltitude() returns arrival runway\'s glideslope altitude abeam the specified position', (t) => {
```

Comment:
Any idea why?

Reply:
No idea why, but the difference is substantial:

<img width="929" alt="Screen Shot 2019-05-21 at 8 28 41 PM" src="https://user-images.githubusercontent.com/3325985/58140953-1b87a300-7c07-11e9-9d68-5a94ea8794ff.png">

Reply:
fixed this.